Automattic/jetpack
# Remove duplicate views: Fix the Stats and Promote with blaze pages

Related to https://github.com/Automattic/wp-calypso/issues/95621

## Proposed changes:

* Fix the Stats by post and Promote with blaze for Simple sites part of the RDV experiment

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply the patch on the sandbox
* Go to edit.php and notice that the stats and Promote with Blaze links are working
